README.md

```diff
@@ -184,32 +160,23 @@ Example:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
 
 👤 **Author1**
 
 - GitHub: [@githubhandle](https://github.com/xandro2021)
 - Twitter: [@twitterhandle](https://twitter.com/xandro2021)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/alejandro-salazar-ba0ba7255/)
-- Mastodon: [@Mastodon] (https://uiuxdev.social/@xandrocastro)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- Mastodon: [Mastodon](https://uiuxdev.social/@xandrocastro)
```

Comment:
- [ ] Kindly add your social media handles here. :arrow_double_up:

Reply:
I couldn't understand your optional suggestion. All my social media links are working properly there, including the extra one from Mastodon. Then, I just add my name in the Author1 Title

Reply:
Ok, I change the pull request description as you have suggested